### PR TITLE
[gardening] Fix unjustified spacing

### PR DESCRIPTION
**Pull request by practicalswift**
#### What's in this pull request?

Fix unjustified spacing.
#### Resolved bug number: –

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

 **Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| OS X platform | @swift-ci Please test OS X platform |
| Linux platform | @swift-ci Please test Linux platform |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

File: test/1_stdlib/VarArgs.swift [hunk 1]
```diff
@@ -85,12 +85,12 @@ func test_varArgs4() {
   let f64 = Double(2.2)
   let fCG = CGFloat(3.3)
 
-  my_printf("a %g %d %g %d %g %d a\n",            f32, i8, f64, i8, fCG, i8)
-  my_printf("b %d %g %d %g %d %g %d b\n",     i8, f32, i8, f64, i8, fCG, i8)
-  my_printf("c %d %d %d %d %d %lld %d c\n",       i8, i16, i8, i32, i8, i64, i8)
-  my_printf("d %d %d %d %d %d %d %lld %d d\n",i8, i8, i16, i8, i32, i8, i64, i8)
-  my_printf("e %u %u %u %u %u %llu %u e\n",       u8, u16, u8, u32, u8, u64, u8)
-  my_printf("f %u %u %u %u %u %u %llu %u f\n",u8, u8, u16, u8, u32, u8, u64, u8)
```
Comment (gribozavr) [inline test/1_stdlib/VarArgs.swift]: This one was better before the change.

**Comment by gribozavr** [comment]
LGTM modulo comments and merge conflicts.

**Comment by practicalswift** [comment]
@gribozavr Fixed :-)

**Comment by practicalswift** [comment]
@swift-ci Please test and merge :-)

**Pull request merged**